keyboards/hhkb/yang/matrix.c

```diff
@@ -0,0 +1,211 @@
+/*
```

Comment:
Would it be possible to convert this to matrix lite?

Reply:
Yes. I was not familiar with the code base. Converting to matrix lite makes the code a lot simplier. Thanks!

Reply:
Not a problem!